app/src/main/java/com/example/makeitso/model/service/User.kt

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright 2022 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+ */
+
+package com.example.makeitso.model.service
+
+data class User(val id: String = "", val isAnonymous: Boolean = true)
```

Comment:
Can you move this file to `model` instead of `service` directory? (`model` is where `Task` and `Priority` are).